src/SimpleDetailsList.tsx

```diff
@@ -87,16 +89,16 @@ export class SimpleDetailsList extends React.Component<any, IDetailsListBasicExa
     return (
       <div className='ms-DetailsListBasicExample'>
         <Toggle
-          isToggled={ isGridVisible }
+          checked={ isGridVisible }

```

Comment:
Notice that we've updated the property here; we're moving to this convention for common things to make it more predictable in usage. Also notice in your project that it looks much prettier. :)